doc/user/ospf6d.rst

```diff
@@ -268,6 +268,15 @@ Showing OSPF6 information
    Interface name can also be given. JSON output can be obtained by appending
    'json' to the end of command.
 
+.. index:: show ipv6 ospf6 spf tree [json]
+.. clicmd:: show ipv6 ospf6 spf tree [json]
+
+   This commands shows the spf tree from the recent spf calculation with the
+   calling router as the root. If json is appended in the end, we can get the
+   tree in JSON format. Each area that the router belongs to has it's own
+   JSON object, with each router having "cost", "isLeafNode" and "childs" as
```

Comment:
I think this should be children not childs as the json output

Reply:
Fixed